src/wp-includes/class-wp-xmlrpc-server.php

```diff
@@ -164,6 +171,48 @@ public function __construct() {
 		 * @param string[] $methods An array of XML-RPC methods, keyed by their methodName.
 		 */
 		$this->methods = apply_filters( 'xmlrpc_methods', $this->methods );
+
+		$this->set_is_enabled();
+	}
+
+	/**
+	 * Set wp_xmlrpc_server::$is_enabled property.
+	 * Determine whether the xmlrpc server is enabled on this WordPress install * and set the is_enabled property accordingly.
+	 * @since 5.7.2
+	 */
```

Comment:
```suggestion
	/**
	 * Set wp_xmlrpc_server::$is_enabled property.
	 *
	 * Determine whether the xmlrpc server is enabled on this WordPress install
	 * and set the is_enabled property accordingly.
	 *
	 * @since 5.7.3
	 */
```